topic-controller/src/main/java/io/strimzi/controller/topic/BackOff.java

```diff
@@ -14,7 +14,11 @@
     private int attempt = 0;
 
     public BackOff() {
-        this(200L, 2, 4);
+        this(200L, 2, 6);
```

Comment:
We should either factor out constants for the `200L` and `2` literals, or we delegate to the `BackOff(int)` constructor. But we should avoid having duplicate literals for these things across 2 ctors.

Reply:
Agree.